lib/iex/lib/iex/autocomplete.ex

```diff
@@ -297,26 +297,48 @@ defmodule IEx.Autocomplete do
       not ensure_loaded?(mod) ->
         []
       docs = Code.get_docs(mod, :docs) ->
-        exports(mod) |> Enum.reject(&hidden_fun?(&1, docs))
+        (exports(mod) -- get_default_arg_functions_with_doc_false(docs))
+        |> Enum.reject(&underscored_fun_without_docs?(&1, docs))
       true ->
         exports(mod)
     end
   end
 
-  defp hidden_fun?(fun, docs) do
+  defp get_default_arg_functions_with_doc_false(docs) do
```

Comment:
Please ignore this. I see that we still need to check for `doc` below regardless. I would maybe just remove the `get_` prefix (they are mostly uncommon in Elixir).